components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -322,13 +302,7 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
                 isCollapsible = false;
             }
         } else {
-            categoryMenu = (
-                <SidebarCategoryMenu
-                    category={category}
-                    isMenuOpen={this.state.isMenuOpen}
-                    onToggleMenu={this.handleMenuToggle}
-                />
-            );
+            categoryMenu = (<SidebarCategoryMenu category={category}/>);
```

Comment:
there is no need for the parantheses anymore when single line

```suggestion
            categoryMenu = <SidebarCategoryMenu category={category}/>;
```

Reply:
changed